build.gradle

```diff
@@ -95,7 +95,7 @@ subprojects {
         jaxbApiVersion = '2.3.1'
         jaxbCoreVersion = '2.3.0.1'
         jaxbImplVersion = '2.3.2'
-        jbcryptVersion = '0.4'
+        bcryptVersion = '0.9.0'
```

Comment:
Looks like someone went to the trouble to sort this list lexicographically; might want to move this entry up.